### PR TITLE
node: Improve signals declaration

### PR DESCRIPTION
**Pull request by galkin**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://nodejs.org/api/deprecations.html#deprecations_dep0008_require_constants

### CODE REVIEW

**Review by SimonSchick — COMMENTED** [review]
I'd suggest you only update v10 and v12 and not bother with updating any unsupported version.

File: types/node/constants.d.ts [hunk 1]
```diff
@@ -133,15 +134,25 @@ declare module "constants" {
     const WSA_E_NO_MORE: number;
     const WSA_E_CANCELLED: number;
     const WSAEREFUSED: number;
+    /** @deprecated since v6.3.0 - use `os.constants.signals.SIGHUP` instead. */
```
[the remark this answers is not in the document]
Comment (galkin) [reply types/node/constants.d.ts]: There are two cases:

1. The first line with deprecation for whole module covers such import:
```typescript
import * as constants from 'constants';
```

2. Deprecation line for every constants covers such import:
```typescript
import { SIGHUP } from 'constants';
```

Without deprecation for every constant we can help linter/IDEA know about deprecation with import from the second case.

**Comment by galkin** [comment]
@Flarna, @SimonSchick, @trivikr, could you provide feedback?

**Review by Flarna — APPROVED** [review]
As indicated already there is no need to update definitions for EOL versions; but it is not wrong to do so.

**Review by trivikr — APPROVED** [review]

**Comment by jessetrinity** [comment]
@galkin I'm checking with the team if it's okay to merge PRs with benchmark failures as we speak :)

**Comment by galkin** [comment]
let's merge this.

**Pull request merged**